lib/logflare/backends/adaptor/clickhouse_adaptor.ex

```diff
@@ -379,22 +357,16 @@ defmodule Logflare.Backends.Adaptor.ClickhouseAdaptor do
 
     event_params =
       Enum.map(events, fn %LogEvent{} = log_event ->
-        flattened_body =
-          log_event.body
-          |> Map.drop(["id", "event_message", "timestamp"])
-          |> Iteraptor.to_flatmap()
-
         [
           log_event.body["id"],
-          log_event.body["event_message"],
-          Jason.encode!(flattened_body),
+          Jason.encode!(log_event.body),
           DateTime.from_unix!(log_event.body["timestamp"], :microsecond)
         ]
       end)
 
     opts = [
-      names: ["id", "event_message", "body", "timestamp"],
-      types: ["UUID", "String", "String", "DateTime64(6)"]
+      names: ["id", "payload", "timestamp"],
```

Comment:
why not preserve the `body` column name? 
it would match the LogEvent's `body` as well as postgres backend's use of `body` column.